### DISCUSSION TITLE
How to make website private for now?

### DISCUSSION THREAD

**Discussion by mozonte** [datenstrom/community#449] [Ask a question] [answered]
How can I require login/password in order to view the website?

**Comment by schulle4u** [comment, marked as answer]
If you are on Apache, you should be able to apply standard authentication rules to the `.htaccess` file in Yellow's root directory. Authentication generators are widely available. Please consult the appropriate documentation For other server software like Nginx. 

To protect individual pages you can try [my private extension](https://github.com/schulle4u/yellow-extensions-schulle4u/tree/master/private).
